meteor/client/lib/splitDropdown.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import * as _ from 'underscore'
+import * as ClassNames from 'classnames'
+
+import * as faChevronUp from '@fortawesome/fontawesome-free-solid/faChevronUp'
+import * as FontAwesomeIcon from '@fortawesome/react-fontawesome'
+
+interface IProps {
+	elements: Array<React.ReactNode>
+	selectedKey: string
+	className?: string
+}
+
+interface IState {
+	expanded: boolean
+}
+
+export class SplitDropdown extends React.Component<IProps, IState> {
```

Comment:
I have only now realized that the file name for this component is `splitDropdown.tsx` and the component name is `SplitDropdown`. In order to keep in line with our naming convention, this should be renamed to `SplitDropdown.tsx`, since it's a React component.